addons/stock/models/product.py

```diff
@@ -599,8 +593,10 @@ class ProductTemplate(models.Model):
     )
     @api.depends_context('company')
     def _compute_quantities(self):
-        res = self._compute_quantities_dict()
-        for template in self:
+        # disable active_test to count also archived products
+        new_self = self.with_context(active_test=False)
+        res = new_self._compute_quantities_dict()
+        for template in new_self:
```

Comment:
it would be nice to add a simple test somewhere showing that qty_available returns the right values for an archived product

Reply:
Test added.

Reply:
@sle-odoo all issues fixed